frontend/src/components/header/GetMore.tsx

```diff
@@ -0,0 +1,75 @@
+import {useState} from "react";
+import axios from "axios";
+import {Form, InputGroup} from "react-bootstrap";
+
+ function GetMore() {
+        const [url, setUrl] = useState("");
+        const [dropdownCategory, setDropdownCategory] = useState("");
+        const [name, setName] = useState("");
+        const [title, setTitle] = useState("");
+
+     function handleSubmit() {
+            const newBookmarkDTO = {
+                bookmarkUrl: url,
+                bookmarkDropdownCategory: dropdownCategory,
+                bookmarkName: name,
+                bookmarkTitle: title
+            }
+
+            axios.post('api/bookmarks/add', newBookmarkDTO)
+                .then((response) => {
+                    console.log('Bookmark added successfully:', response.data);
```

Comment:
solche ausgaben gehören in einen logger, aber nicht in die konsole :)

Reply:
Ist jetzt erstmal raus gekommen.